Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -64,12 +67,18 @@ public final class ProjectPamphletViewModel: ProjectPamphletViewModelType, Proje
           }
       }
 
-    self.goToRewards = freshProjectAndRefTag
+    let goToRewards = freshProjectAndRefTag
       .takeWhen(self.backThisProjectTappedProperty.signal)
       .map { project, refTag in
         (project, refTag)
       }
 
+    self.goToRewards = goToRewards
```

Comment:
Just curious, why not convert this signal into one that takes a `isDeprecated` bool: `public let goToRewards: Signal<(Project, RefTag?, Bool), Never>`. It's a little confusing that there are two separate signals 🤔

Reply:
Easier to delete later?